script/core.hbs

```diff
@@ -103,7 +103,8 @@
         window.iframeLoaded.then(() => {
           {{> script/iframe-messaging}}
         });
-      }
+      },
+      ...{{#stripJsComments}}{{> script/custom-init-config }}{{/stripJsComments}}
```

Comment:
Should we wrap all of `onReady` in this helper?

Reply:
I was thinking it wouldn't hurt to let them override onReady if they wanted, but at that point you should probably override core.hbs